_data/toc.yaml

```diff
@@ -157,6 +155,10 @@ guides:
       title: Manage data in containers
   - path: /engine/docker-overview/
     title: Docker overview
+  - path: /release-notes/docker-ce/
```

Comment:
Please don't put Release Notes under "Getting Started"

Why not just make "Release Notes" a node under Guides and have these be sub-topics?